File: Cappacita/ex_11.js
// Faça um loop que procure no array doces a palavra pirulito e troque por doce de
// leite

doces = ["bala", "pirulito", "chiclete", "jujuba", "chocolate"]

console.log("Antes: " + doces)
for (doce in doces){
    if(doces[doce] == "pirulito"){
        doces[doce] = "doce de leite"
    }   
}
console.log("Depois: " + doces)

File: Cappacita/ex_11.py
doces = ["bala", "pirulito", "chiclete", "jujuba", "chocolate"]

print(f'Antes: {doces}')
for num, doce in enumerate(doces):
    if doces[num] == "pirulito":
        doces[num] = "doce de leite"    
print(f'Depois: {doces}')

